python/ray/_raylet.pyx

```diff
@@ -667,6 +667,8 @@ cdef execute_task(
                 errors.append(failure_object)
             core_worker.store_task_outputs(
                 worker, errors, c_return_ids, returns)
+            logger.warning("Failed to execute the task({}) with the error:{}"
```

Comment:
should use `logger.error`?

Reply:
why not show `errors` that contains all failure object

Reply:
`errors` is used to put into the object in plasma.